src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -115,6 +115,7 @@ public class StripePaymentService : IPaymentService
         Stripe.Subscription subscription;
         try
         {
+            stripeCustomerMetadata.Add("organization", org.Name);
```

Comment:
It looks like you're including the same information twice, once in the customer metadata and once in the invoice settings. Are both necessary? It's fine if it is, I just don't know how it works on the Stripe end so I wanted to check.